casr/src/bin/casr-cli.rs

```diff
@@ -447,6 +447,16 @@ fn build_tree_report(
         tree.collapse_item(row);
     }
 
+    if !report.js_report.is_empty() {
```

Comment:
Update [here](https://github.com/ispras/casr/blob/09b003ed91971f79ca18cc38ae3d9816d7d7b6d1/libcasr/src/report.rs#L581)

Reply:
It's already updated there. Or do I need something else?